cmd/cluster-init/update_sanitize_prow_jobs.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/dispatcher"
+	"github.com/openshift/ci-tools/pkg/jobconfig"
+)
+
+func updateSanitizeProwJobs(o options) error {
+	logrus.Info("Updating sanitize-prow-jobs config")
+	filename := filepath.Join(o.releaseRepo, "core-services", "sanitize-prow-jobs", "_config.yaml")
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+	var c dispatcher.Config
+	if err = yaml.Unmarshal(data, &c); err != nil {
+		return err
+	}
+	updateConfig(&c, o.clusterName)
+	rawYaml, err := yaml.Marshal(c)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(filename, rawYaml, 0644)
+}
+
+func updateConfig(c *dispatcher.Config, clusterName string) {
+	appGroup := c.Groups[api.ClusterAPPCI]
+	metadata := api.Metadata{
+		Org:    "openshift",
+		Repo:   "release",
+		Branch: "master",
+	}
+	appGroup.Jobs = append(appGroup.Jobs, metadata.JobName(jobconfig.PresubmitPrefix, clusterName+"-dry"))
+	appGroup.Jobs = append(appGroup.Jobs, metadata.JobName(jobconfig.PostsubmitPrefix, clusterName+"-apply"))
+	appGroup.Jobs = append(appGroup.Jobs, fmt.Sprintf("%s-%s-%s-%s-%s-apply",
```

Comment:
Why is this not using the `JobName()` helper? Can we add a periodic helper if the format is different?

Reply:
The helper adds "ci" in there, which we don't want for periodic. I can add a new helper though.